src/uv.cc

```diff
@@ -60,15 +60,13 @@ using v8::Value;
 
 void ErrName(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
-  if (env->options()->pending_deprecation && env->EmitErrNameWarning()) {
-    if (ProcessEmitDeprecationWarning(
-        env,
-        "Directly calling process.binding('uv').errname(<val>) is being"
-        " deprecated. "
-        "Please make sure to use util.getSystemErrorName() instead.",
-        "DEP0119").IsNothing())
-    return;
-  }
+  if (ProcessEmitDeprecationWarning(
+      env,
+      "Directly calling process.binding('uv').errname(<val>) is being"
+      " deprecated. "
+      "Please make sure to use util.getSystemErrorName() instead.",
```

Comment:
While we're in here, can we change "Please make sure to use" to "Use" or "Please use"?

Reply:
Sure!